static/js/pivotal.js

```diff
@@ -42,29 +53,41 @@
     }
   }
 
+  /**
+   * getProjectDiffs return an Object, project name as the key and github diff URLs
+   * as the value
+   * @return {Object} {project_name: diff_url}
+   */
   function getProjectDiffs() {
-    var project_diffs = [];
+    var diffURLs = {};
 
     $(config.selectors.project).each(function() {
-      var project = $(this).attr(config.selectors.project_id);
-      var url = $(this).find(config.selectors.github_link).attr('href');
-      if (url && url != 'undefined') {
-        project_diffs[project] = url;
+      var $this = $(this);
+      var project = $this.data('id');
+      var url = $this.find(config.selectors.github_link).attr('href');
+
+      if (url) {
+        diffURLs[project] = url;
       }
     });
 
-    return project_diffs;
+    return diffURLs;
   }
 
+  /**
+   * getLinkDiv return an HTML string of story link and status
+   * @param  {Object} pt_info Pivotal story object
+   * @return {String}         HTML string
+   */
   function getLinkDiv(pt_info) {
     var links = [];
-    var info;
-    var status_label;
 
     for (f in pt_info) {
```

Comment:
@patatepartie Tried it, but as `pt_info` is an `Object` can't use `Array.map` :cry: